scripts/init.js

```diff
@@ -4,7 +4,12 @@ const path = require('path')
 const validateConfig = require('../lib/validateConfig')
 const { Red, Yellow, Green } = require('../lib/colors')
 
-module.exports = () => {
+const defaultOptions = {
+  skipFailure: false,
+  exitAfterExecute: true,
```

Comment:
If we want to use scripts as just independent functions maybe we need to make them as such. I mean we can make them return promises and call .then(() => process.exit(0), () => process.extit(1)) in the index.js.